README.md

```diff
@@ -53,6 +53,21 @@ Utility for serially testing and merging pull requests in conjunction with Jenki
 
  * Run `test_pull_requests` as a Jenkins or `cron` job.  Note that GitHub is rate limited but typical projects can still run this script every few mins without running out of requests.
 
+### Setup for merge_queue_overview
+ * Add `merge_queue_overview` to your Jenkins system and make sure it's executable
+ * Add `.merge_queue_overview.json` to `$JENKINS_HOME`
```

Comment:
I don't love the name with _for for the one we are using either.  Also, if we have a file per set of config, let's add a name to it so you don't see test_pull_requests in the html name.